src/main/java/org/folio/rest/delegate/EmailDelegate.java

```diff
@@ -53,21 +55,26 @@ public void execute(DelegateExecution execution) throws Exception {
 
     String subjectTemplate = this.mailSubject.getValue(execution).toString();
     String textTemplate = this.mailText.getValue(execution).toString();
-
-    Map<String, Object> inputs = getInputs(execution);
-
-    Configuration cfg = new Configuration(Configuration.VERSION_2_3_23);
+    String markupTemplate = Objects.nonNull(this.mailMarkup) ? this.mailMarkup.getValue(execution).toString() : "";
```

Comment:
I'd rather not duplicate the "email body", text, variable. Can we not refactor text to body or something less imposing type and create another property, either boolean or enum, that represents text or HTML?

Reply:
I think `boolean asHTML`  would suffice. Using https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/mail/javamail/MimeMessageHelper.html#setText-java.lang.String-boolean- seems appropriate. I am not sure though. Maybe we keep the markup as an alternative. If so, we should just use setText(text, false) if it is not intended to be html.

Reply:
It needs to support both.
The e-mail uses separate fields for plain text and markup.

It is better to build the format as needed in the workflow rather than expect the service to know how to convert HTML markup into text.

That said, we could do that.
The logic would be simple except that it would be less flexible.
That said, there would only need to be markup transmitted via the workflow.
Then the service would construct two fields, a plain text and a markup.

Reply:
Nevermind you implemented more logic then I noticed.